### PR TITLE
added docker file to run the smoke tests in jenkins

### PR DESCRIPTION
**Pull request by satyachundur**

### CODE REVIEW

File: bin/runsmoketest.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  docker-compose -f docker-compose.smoke-tests.yml down
+}
+
+
+trap shutdownDocker INT TERM QUIT EXIT
+
+docker-compose --version
+
+
+docker-compose -f docker-compose.smoke-tests.yml run syatests
```
Comment (bartlabno) [inline bin/runsmoketest.sh]: Will it run properly without `docker-compose -f docker-compose.smoke-tests.yml build` first?

Comment (satyachundur) [reply bin/runsmoketest.sh]: We are not building docker image here, as we using existing image to reform/codeceptjs to run our tests against deployed instance

File: test/e2e/page-objects/hearing/theHearing.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 function enterDoYouWantToAttendTheHearing(option) {
 
     const I = this;
-
```
Comment (paulgain) [inline test/e2e/page-objects/hearing/theHearing.js]: For consistency throughout all of out tests please put the space back.

File: test/e2e/page-objects/start/postcode-checker.js [hunk 3]
```diff
@@ -1,7 +1,6 @@
 function enterPostcodeAndContinue(postcode) {
 
     const I = this;
-
     I.fillField('#postcode', postcode);
```
Comment (paulgain) [inline test/e2e/page-objects/start/postcode-checker.js]: Same as above.

File: test/e2e/page-objects/hearing/arrangements.js [hunk 4]
```diff
@@ -5,7 +5,6 @@ const fields = require('steps/hearing/arrangements/content.en.json').fields;
 function checkAllArrangementsAndContinue() {
 
     const I = this;
-
```
Comment (paulgain) [inline test/e2e/page-objects/hearing/arrangements.js]: For consistency throughout all of our tests please put the space back.

**Pull request merged**